### ISSUE TITLE
Wrong colour for keyword "length" in keyword list

### ISSUE THREAD

**Issue by brianvanburken** [elixir-lang/vim-elixir#206]
When adding a keyword "length" it gets coloured in red. 
Expected colour is same as keyword above.

![screen shot 2016-10-28 at 10 42 32](https://cloud.githubusercontent.com/assets/1044316/19800342/9505efe0-9cfb-11e6-8e07-1544715806b0.png)

**Comment by jbodah** [comment]
Hi, are you using the latest master? The following test is passing for me:

```rb
# frozen_string_literal: true

require 'spec_helper'

describe 'Keyword syntax' do
  it 'a kernel function used as an atom key in a keyword list' do
    expect(<<~EOF).to include_elixir_syntax('elixirAtom', 'length')
    plug Plug.Parsers,
      parsers: [:urlencoded, :multipart, :json],
      pass: ["*/*"],
      json_decoder: Poison,
      length: 400_000_000
    EOF
  end
end
```

**Comment by brianvanburken** [comment]
I've checked and yes I'm using the latest master. According to the local repo I'm on commit d3d964a8b1953e12cdadb77b3f255923f4873637 for master. I've tried to debug if other plugins are causing this but no result. When only enabling the `vim-elixir` the problem still exists. So far I'm able to reproduce this on my machine.

This is my .vimrc file: https://github.com/brianvanburken/dotfiles/blob/master/.vimrc
The file I'm testing the problem with: https://github.com/brianvanburken/elixir-playground/blob/master/phoenix_video_stream/lib/phoenix_video_stream/endpoint.ex

My Vim info:

```
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Oct 22 2016 08:48:49)
MacOS X (unix) version
Included patches: 1-46
Compiled by Homebrew
Huge version without GUI.  Features included (+) or not (-):
+acl             -ebcdic          +listcmds        +persistent_undo +textobjects
+arabic          +emacs_tags      +localmap        +postscript      +timers
+autocmd         +eval            -lua             +printer         +title
-balloon_eval    +ex_extra        +menu            +profile         -toolbar
-browse          +extra_search    +mksession       +python          +user_commands
++builtin_terms  +farsi           +modify_fname    -python3         +vertsplit
+byte_offset     +file_in_path    +mouse           +quickfix        +virtualedit
+channel         +find_in_path    -mouseshape      +reltime         +visual
+cindent         +float           +mouse_dec       +rightleft       +visualextra
-clientserver    +folding         -mouse_gpm       +ruby            +viminfo
+clipboard       -footer          -mouse_jsbterm   +scrollbind      +vreplace
+cmdline_compl   +fork()          +mouse_netterm   +signs           +wildignore
+cmdline_hist    -gettext         +mouse_sgr       +smartindent     +wildmenu
+cmdline_info    -hangul_input    -mouse_sysmouse  +startuptime     +windows
+comments        +iconv           +mouse_urxvt     +statusline      +writebackup
+conceal         +insert_expand   +mouse_xterm     -sun_workshop    -X11
+cryptv          +job             +multi_byte      +syntax          -xfontset
+cscope          +jumplist        +multi_lang      +tag_binary      -xim
+cursorbind      +keymap          -mzscheme        +tag_old_static  -xpm
+cursorshape     +lambda          +netbeans_intg   -tag_any_white   -xsmp
+dialog_con      +langmap         +num64           -tcl             -xterm_clipboard
+diff            +libcall         +packages        +termguicolors   -xterm_save
+digraphs        +linebreak       +path_extra      +terminfo
-dnd             +lispindent      +perl            +termresponse
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
  fall-back for $VIM: "/usr/local/share/vim"
Compilation: clang -c -I. -Iproto -DHAVE_CONFIG_H   -DMACOS_X_UNIX  -g -O2 -U_FORTIFY_SOU
RCE -D_FORTIFY_SOURCE=1
Linking: clang   -L. -fstack-protector -L/usr/local/lib -L/usr/local/opt/libyaml/lib -L/u
sr/local/opt/openssl/lib -L/usr/local/opt/readline/lib  -L/usr/local/lib -o vim        -l
ncurses -liconv -framework Cocoa   -mmacosx-version-min=10.12 -fstack-protector-strong -L
/usr/local/lib  -L/usr/local/Cellar/perl/5.24.0_1/lib/perl5/5.24.0/darwin-thread-multi-2l
evel/CORE -lperl -lm -lutil -lc -F/usr/local/opt/python/Frameworks -framework Python   -l
ruby.2.3.0 -lobjc -L/usr/local/Cellar/ruby/2.3.1_2/lib
```

If you need more information let me know!

**Comment by jbodah** [comment]
figured it out, I had to wrap it in a `defmodule` should be fixed with #207

**Comment by brianvanburken** [comment]
❤️ 💚 💙 💛 💜
